Makefile

```diff
@@ -8,9 +8,11 @@ ALL_V_WITHOUT_TESTS_OR_CODEGEN := $(shell \
 		! -path '*/test/*' \
 		! -name '*.gen.v' \
 )
+CLASH_VERILOG = circuit/execute/alu.v
```

Comment:
Small thing, but maybe embed in `ALL_V`?

Something like:

```make
ALL_V := $(ALL_V) $(CLASH_VERILOG)
```

...assuming that works and doesn't break the other targets 🙂